test/tools/Modules/HttpListener/HttpListener.psm1

```diff
@@ -134,6 +134,13 @@ Function Start-HTTPListener {
                             $contentType = $queryItems["contenttype"]
                             $output = $queryItems["output"]
                         }
+
+                        # Echo the request as the output.
+                        "echo"
```

Comment:
I think it makes more sense to have a separate 'echo' test.  The returned data for response would be very different; it currently doesn't echo the request, and using contentype to control that doesn't make sense to me.